ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -55,6 +55,9 @@ chain:
           echo "$(date --rfc-3339=seconds) Clusteroperators not yet ready"
           sleep 1s
       done
+      # Data for cluster bot.
+      echo "https://$(oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > "${SHARED_DIR}/console.url"
+      cp ${SHARED_DIR}/nested_kubeconfig ${SHARED_DIR}/kubeconfig
```

Comment:
Can we still cleanup the hostedcluster we just created when we override the mgmt cluster kubeconfig?

Reply:
good point, let me update.

Reply:
added a check to pick the manament cluster kubeconfig into hypershift-aws-destroy-chain